test/Conversion/amd/mfma-shortcut.mlir

```diff
@@ -0,0 +1,28 @@
+// XFAIL: *
```

Comment:
This test is expect to fail, because `kWidth` parameter is not parsed correctly.
Will be fixed in #3676